celery-stubs/worker/request.pyi

```diff
@@ -137,7 +138,7 @@ class Request:
     def reject(self, requeue: bool = ...) -> None: ...
     def info(self, safe: bool = ...) -> _InfoDict: ...
     def humaninfo(self) -> str: ...
-    def chord(self) -> Any | None: ...
+    def chord(self) -> chord_type: ...
```

Comment:
actually not sure on this one. should it be optional[chord_type]?

Reply:
Hmm I think looking at the code it should be `Optional[chord_type]` aka `chord_type | None`

https://github.com/celery/celery/blob/018f996fe2f80ed2e69ffd93bd9b35dfa41d1141/celery/worker/request.py#L665

Reply:
thanks, updated